lib/Service/LockService.php

```diff
@@ -126,7 +111,7 @@ public function __construct(
 	 *
 	 * @return FileLock|bool
 	 */
-	private function getLockForNodeId(int $nodeId) {
+	public function getLockForNodeId(int $nodeId) {
 		if (array_key_exists($nodeId, $this->lockCache) && $this->lockCache[$nodeId] !== null) {
```

Comment:
I remember seeing in some place that we prefer `isset()` to `array_key_exists()`. Not a big deal though.